web/src/compose_validate.js

```diff
@@ -245,6 +246,22 @@ export function warn_if_topic_resolved(topic_changed) {
     }
 }
 
+export function warn_if_in_search_view() {
+    if (narrow_state.filter()?.is_search()) {
```

Comment:
`filter.is_search` is the wrong check. Can you do a quick little PR to rename that to `is_keyword_search`? I feel like it trips up a lot of people in this way -- the function just checks if there's a keyword, not if there are search operators.

You want `filter.supports_collapsing_recipients()` instead.

Reply:
Have fixed the if condition in #27864